pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -222,10 +222,15 @@ func Convert_v1_Disk_To_api_Disk(c *ConverterContext, diskDevice *v1.Disk, disk
 		volumeStatus, ok := volumeStatusMap[diskDevice.Name]
 		if ok && volumeStatus.PersistentVolumeClaimInfo != nil {
 			disk.FilesystemOverhead = volumeStatus.PersistentVolumeClaimInfo.FilesystemOverhead
-			capacity, ok := volumeStatus.PersistentVolumeClaimInfo.Capacity[k8sv1.ResourceStorage]
+			storageCapacity, ok := volumeStatus.PersistentVolumeClaimInfo.Capacity[k8sv1.ResourceStorage]
 			if ok {
-				disk.Capacity = &capacity
+				disk.StorageCapacity = &storageCapacity
 			}
+			storageRequest, ok := volumeStatus.PersistentVolumeClaimInfo.Requests[k8sv1.ResourceStorage]
+			if ok {
+				disk.StorageRequest = &storageRequest
+			}
+
```

Comment:
Since we're storing the requests as quantities here, maybe it makes sense to perform the min check here rather than in manager.go? So the disk would only be carrying the actual usuable capacity instead of both quanities for the PVC requested capacity and status.Capacity.

Reply:
CDI has already created a function to calculate the minimum of two quantities: https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/util/util.go#L149-L155

Reply:
Doing this in new version.